nesta/core/batchables/nlp/vectorizer/run.py

```diff
@@ -49,8 +49,10 @@ def run():
 
     # Extract what you need from the data
     _data = [list(itertools.chain.from_iterable(row[text_field]))
-             for row in data if type(row[text_field]) is list]
+             if type(row[text_field]) is list
```

Comment:
If it's not a list, then what is this defending against? A string?

Reply:
This block would benefit from a little documentation and could perhaps be in a function.

Reply:
Thanks - I've added comments, docs, a function, and testing.